### DISCUSSION TITLE
How to show the edit image modal directly when the image is uploaded for fixed aspect ratio selector?

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#11874] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

```php
                SpatieMediaLibraryFileUpload::make('image')
                    ->imageEditor()
                    ->imageEditorAspectRatios([
                        '16:9',
                    ])
                    ->downloadable()
```

When the user uploads an image, it should open the model for editing and force the 16:9 aspect ratio selector, that way I know when the image is uploaded it is a 16:9 aspect ratio image and users can select their own 16:9 area of the image. Forcing the editor on image upload for the 16:9 selector, this is what I want to do, how do I do that?

Currently, It just uploads the image, and users have the option whether they want to edit or not, I want to force edit so all the images that users upload will have a 16:9 aspect ratio but users can select which 16:9 area they want to select from the image.

**Comment by arbet** [comment]
have you solved this?
